auto3dseg/algorithm_templates/dints/scripts/train.py

```diff
@@ -32,9 +33,55 @@
 from monai.data import DataLoader, partition_dataset
 from monai.inferers import sliding_window_inference
 from monai.metrics import compute_meandice
+from monai.transforms import (
+    apply_transform,
+    Compose,
+    Randomizable,
+    Transform,
+)
 from monai.utils import set_determinism
 
 
+class DuplicateCacheDataset(monai.data.CacheDataset):
+    def __init__(self, times: int, **kwargs):
+        super().__init__(**kwargs)
+        self.times = times
+
+    def __len__(self):
+        return self.times * super().__len__()
+
+    def _transform(self, index: int):
```

Comment:
the CacheDataset now has a `hash_as_key: bool` argument to avoid duplicated item, maybe it's useful here...

https://github.com/Project-MONAI/MONAI/blob/5200cc6e687910118198933b2b6de4ecf97bb881/monai/data/dataset.py#L775-L777